src/wp-includes/widgets/class-wp-widget-media-gallery.php

```diff
@@ -248,6 +248,7 @@ public function render_control_template_scripts() {
 	protected function has_content( $instance ) {
 		if ( ! empty( $instance['ids'] ) ) {
 			$attachments = wp_parse_id_list( $instance['ids'] );
+			_prime_post_caches( $attachments, false, false );
```

Comment:
```suggestion
			// Prime attachment post caches.
			_prime_post_caches( $attachments, false, false );
```

Not blocker. One description would be great.